packages/material-ui-utils/src/debounce.js

```diff
@@ -0,0 +1,18 @@
+// Corresponds to 10 frames at 60 Hz.
+// A few bytes payload overhead when lodash/debounce is ~3 kB and debounce ~300 B.
+export default function debounce(func, wait = 166) {
+  let timeout;
+  function debounced(...args) {
+    const later = () => {
+      func.apply(this, args);
```

Comment:
Any reason why we do not `timeout = undefined` here? AFAICT, the `setTimeout` return value is a `number` in browsers, and these could be recycled.

Reply:
Happy to submit a PR if this makes sense.